CMakeLists.txt

```diff
@@ -530,6 +530,7 @@ if (_M_X86)
   add_subdirectory(Externals/Bochs_disasm)
 endif()
 add_subdirectory(Externals/cpp-optparse)
+add_subdirectory(Externals/fmt EXCLUDE_FROM_ALL)
```

Comment:
Wondering if this should be in the 2nd commit, so that fmt alone is in the first one? Doesn't matter too much I guess, though.